src/content/docs/apm/apm-ui-pages/monitoring/external-services/external-services-intro.mdx

```diff
@@ -0,0 +1,55 @@
+
+The external services feature is no longer just for New Relic APM agent users. It has expanded to include OpenTelemetry and now uses the power of distributed tracing to give you insights into service activity. The classic external services feature is still available behind a UI toggle, but we recommend you try out the full power the expanded external services.  
+
+This feature provides a combination of maps and graphs to help you:
+
+* Find sources of latency and errors
+* Assess the impact of latency to upstream callers
+* Review transaction-level performance
+* Find traces that involve the transactions you're interested in
+
+To use the external services feature, you need a New Relic account with an installed APM agent with distributed tracing enabled. If you don't already have an account, you can sign up [here](https://newrelic.com/signup) for a free account.
+
+![Screenshot showing the opening map for external services](./images/intro-example.png "Screenshot showing the opening map for external services")
+<figcaption>The thickness of the lines represents the throughput from your service to the upstream or downstream services. When you select a specific service, you will see the various endpoints making calls between the two services.</figcaption>
+
+## Relationship to classic external services [#classic-external-services]
+
+While you can still reach classic external services by using the UI toggle, the main external services UI is now populated by data from distributed tracing. 
+It still provides transaction data similar to classic external services, but here are some key things you need to know about the expanded external services: 
+
+  * **Dependency:** To use the external services feature, you need to enable distributed tracing on services that make calls to each other. 
+  * **Compatibility:** Distributed tracing is not backwards compatible with cross-application tracing, so if you currently rely on classic external services, please note that you will only have visibility into calls between services using the same protocol. 
+  * **Data:** Unlike classic external services, the transaction-level detail of distributed tracing is sampled instead of based on metrics. This sampled data links into distributed tracing, which can give you deeper insights into what’s driving the performance of the transactions. 
+
+
+<Callout variant="tip">
+Browser and mobile data is not currently supported in the external services feature.
+</Callout>
+
+## When would you use external services? [#when-to-use]
+
+The external services feature is a tool you can use by itself to tune or troubleshoot a specific service, or you can use it as a starting point for additional troubleshooting with distributed tracing or even infrastructure agents. 
```

Comment:
Maybe break into 2 sentences at the comma? "...service. You can also use it..."?
Should this be "infrastructure monitoring agents"?

Reply:
@barbnewrelic, I broke this into two sentences.